site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/controller/SiteController.java

```diff
@@ -0,0 +1,40 @@
+package uk.ac.ox.ndph.mts.site_service.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.ac.ox.ndph.mts.site_service.model.Site;
+import uk.ac.ox.ndph.mts.site_service.service.SiteService;
+
+/**
+ * Controller for site endpoint of site-service
+ */
+@RestController()
+public class SiteController {
+
+    private static final String ENDPOINT_PATH = "/sites";
```

Comment:
endpoint is site not sites

Reply:
It needs to be "sites" per Kate's PR where we understood that the common naming convention should be plural.

Reply:
This has been fixed using 
@RequestMapping("/sites")
and
@PostMapping()